scripts/build.sh

```diff
@@ -35,6 +35,7 @@ mkdir -p bin/
 echo "==> Building..."
 set +e
 gox \
+    -cgo \
```

Comment:
@cbednarski Changed to user env var ```$USER```. If you have a win machine can you test that env var ```%USER%``` is set to your username? (I only have posix machines available)